Algorithm.Framework/Selection/OpenInterestFutureUniverseSelectionModel.cs

```diff
@@ -95,7 +95,7 @@ public IEnumerable<Symbol> FilterByOpenInterest(IReadOnlyDictionary<Symbol, Secu
         {
             var current = _algorithm.UtcTime;
             var endTime = Instant.FromDateTimeUtc(_algorithm.UtcTime).InZone(exchangeHours.TimeZone).ToDateTimeUnspecified();
-            var previousDay = Time.GetStartTimeForTradeBars(exchangeHours, endTime, Time.OneDay, 1, true);
+            var previousDay = Time.GetStartTimeForTradeBars(exchangeHours, endTime, Time.OneDay, 1, true, exchangeHours.TimeZone);
```

Comment:
`exchangeHours.TimeZone` isn't the data timezone but the exchange time zone. Instead of sending in the `SecurityExchangeHours` above, we could send the entire entry which has the data time zone